presto-main/src/main/java/com/facebook/presto/operator/PagesIndex.java

```diff
@@ -185,6 +187,12 @@ public LongArrayList getValueAddresses()
         return valueAddresses;
     }
 
+    @VisibleForTesting
+    public ObjectArrayList<Block>[] getChannels()
```

Comment:
Is this still necessary? Don't see it called anywhere in the rest of the diff

Reply:
Thanks for finding this